cmd/microcloud/add.go

```diff
@@ -68,10 +68,6 @@ func (c *cmdAdd) Run(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Failed to get MicroCloud status: %w", err)
 	}
 
-	if !status.Ready {
-		return fmt.Errorf("MicroCloud is uninitialized, run 'microcloud init' first")
```

Comment:
In case MicroCloud is uninitialized, should we still add the hint to run `microcloud init` in the new error message?

Reply:
I've added a check on the CLI side that detects if the service is MicroCloud, and if we are not in preseed mode, it hints to use either `microcloud init` or `microcloud add` according to the expectation.

Reply:
Thanks!